build.sbt

```diff
@@ -81,8 +81,8 @@ val caffeineVersion = "2.8.1"
 val bigtableClientVersion = "1.8.0"
 val generatedGrpcGaVersion = "1.43.0"
 val generatedGrpcBetaVersion = "0.44.0"
-val googleClientsVersion = "1.27.0"
-val googleApiServicesBigQuery = s"v2-rev20181104-$googleClientsVersion"
+val googleClientsVersion = "1.28.0"
+val googleApiServicesBigQuery = s"v2-rev20181221-$googleClientsVersion"
 val bigdataossVersion = "1.9.16"
```

Comment:
Why this is 1.9.16? When the latest is 2.0.0?

Reply:
Hi @ChethanUK, we use that version to be inline with `apache/beam`.

Reply:
Okay, I just asked bcoz by default all other products we use (Dataproc, SparkOnK8s, Flink) use 2.0.0.

Reply:
yeah I understand ... `apache/beam` sometimes just takes it's time to update deps.